lib/file_validators/validators/file_size_validator.rb

```diff
@@ -13,16 +13,14 @@ def self.helper_method_name
       end
 
       def validate_each(record, attribute, value)
-        value = JSON.parse(value) if value.is_a?(String) && value.present?
-        unless value.blank?
```

Comment:
This is an optimization feature. If value isn't present the following block won't be executed.